src/app/donation-start/donation-start-match-confirm-dialog.html

```diff
@@ -6,9 +6,9 @@ <h2 mat-dialog-title>{{ data.title }}</h2>
   <p>{{ data.statusDetail }}</p>
 
   <p>Donations received once the charity has hit its target may be used for the charity's general funds.</p>
-  <!-- Temporarily fixed to CC19 copy. This will be pulled from the API soon. -->
-  <p>If the charity has hit their target and is participating in the Christmas Challenge, unmatched
-    donations can help them to win a Christmas Challenge Award.</p>
+
+  <p *ngIf="data.surplusDonationInfo">{{ data.surplusDonationInfo }}</p>
```

Comment:
I've changed 'prepended' to 'appended' in the API field description to reflect what is actually happening here. Based on the overall copy we landed on for CC19 I think this way round makes sense.